### PR TITLE
Addition of Contribute Page

### PR DESCRIPTION
**Pull request by AbhayTopno**
If there are any issues regarding this PR, feel free to reach out to me

### CODE REVIEW

File: README.md
Comment (arkid15r) [inline README.md:1]: Why these changes?

File: backend/data/copy.json.gz
Comment (arkid15r) [inline backend/data/copy.json.gz:1]: Please revert this

File: frontend/src/lib/types.ts [hunk 1]
```diff
@@ -1,3 +1,90 @@
+export interface ContributeDataType {
+  projects: ProjectDataType
+  issues: IssuesDataType
+  chapters: ChapterDataType
+  committees: CommitteeDataType
+}
+
+export const contributeMockData: ContributeDataType = {
```
Comment (arkid15r) [inline frontend/src/lib/types.ts:8]: This should be in the `__tests__/data` directory

**Review by arkid15r — CHANGES_REQUESTED** [review]
Could you add tests similar to other pages?

Thank you!

File: README.md
Comment (AbhayTopno) [reply README.md:1]: It was a space bar added to it

File: backend/data/copy.json.gz
Comment (AbhayTopno) [reply backend/data/copy.json.gz:1]: Ok

File: frontend/src/lib/types.ts (hunk 1 above)
Comment (AbhayTopno) [reply frontend/src/lib/types.ts:8]: I'll make the changes
